src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  public shape: 'triangle' = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
```suggestion
    public color: Color,
```
Consider creating the type for the color instead of duplicating values further in the code